src/common/models/paying-account.model.ts

```diff
@@ -0,0 +1,38 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+import { PayingAccountType } from '@polymeshassociation/polymesh-sdk/types';
+
+import { FromBigNumber } from '~/common/decorators/transformation';
+
+export class PayingAccountModel {
+  @ApiProperty({
+    type: 'string',
+    description: 'The balance of the paying account',
+    example: '29996999.366176',
+  })
+  @FromBigNumber()
+  readonly balance: BigNumber;
+
+  @ApiProperty({ type: 'string', description: 'Paying account type', example: 'Caller' })
+  @ApiProperty({
+    description: 'Paying account type',
+    enum: PayingAccountType,
+    example: PayingAccountType.Caller,
+  })
+  @FromBigNumber()
+  readonly type: string;
+
+  @ApiProperty({
+    type: 'string',
+    description: 'Paying account address on the chain',
```

Comment:
```suggestion
    description: 'The paying account address',
```
(style nit) I think "The" makes it bit more consistent.
(technical nit). The address is an encoding of a private key, so I think an address can "exist" off the chain. OK to leave that part in if you think otherwise